expenses-manager-frontend/src/main/frontend/src/app/login/login.component.scss

```diff
@@ -0,0 +1,77 @@
+.b-login {
+    background-image: url("../../assets/images/background.svg");
+    
+    @mixin login-layout {
+        height: 100%;
+        width: 100%;
+        margin: 0px;
+      
+        background-position: center;
+        background-repeat: no-repeat;
+        background-size: cover;
+    }
+    @include login-layout;
+
+    .login-overlay{
+        background-image: url("../../assets/images/overlay.svg");
+
+        position: absolute;
+
+        @include login-layout;
+    }
+
+
+    .logo{
+        position: relative;
+        top: 200px;
+        width:100%;
+
+        display: flex;
+        flex-direction: column;
+        align-items: center;
+
+        .logo-icon{
+            width: 80px;
+            height: 80px;
+            position: relative;
+        }
+
+        .logo-text{
+
+            font-family: 'Outfit';
+            font-size: 64px;
+            position: relative;
+
+            color: #4F46E5;
+
+        }
+    }
+
+    .login-form{
+        position: relative;
+        top: 200px;
+        width: 100%;
+        margin-top: 30px;
+
+        display: flex;
+        flex-direction: column;
+        align-items: center;
+
+        .username-input{
+            width: 325px;
```

Comment:
O practica modera este folosirea inline-size si block-size in locul lui width si height. 
Se integreaza mai bine cu grid si flexbox.